src/gpu/intel/jit/conv/config.cpp

```diff
@@ -860,7 +860,8 @@ status_t init_tensor_layouts(
         // padding at the end is the easiest way to avoid that, as 1-2 KB of
         // additional VRAM per precalc buffer is virtually free
         // TODO: vectorize send params (in jit:ir:v2 maybe?) and add tmasks!
-        const dim_t max_read_blk_bytes = 512;
+        const dim_t max_read_blk_bytes
+                = (cfg.hw() < ngen::HW::Xe3) ? 512 : 2048;
```

Comment:
From the kernel side Xe3 and Xe2/XeHPC are almost the same. Do you have an explanation why Xe3 is different here?
If it's unclear - better to use the same padding for all platforms.

Reply:
OK, will do.

Reply:
Done.